ftag/tests/wps/test_discriminant.py

```diff
@@ -202,6 +202,21 @@ def test_get_discriminant():
     expected = ctag_discriminant(jets, tagger, fb=0.2)
     assert np.allclose(disc, expected)
 
+    jets = np.array(
+        [
+            (0.2, 0.3, 0.5),
+            (0.8, 0.5, 0.1),
+            (0.6, 0.1, 0.7),
+        ],
+        dtype=[("tagger_pb", "f4"), ("tagger_pc", "f4"), ("tagger_pu", "f4")],
+    )
+
+    tagger = "tagger"
+    signal = Flavours.ghostbjets
+    disc = get_discriminant(jets, tagger, signal, fc=0.1)
+    expected = btag_discriminant(jets, tagger, fc=0.1)
+    assert np.allclose(disc, expected)
+
```

Comment:
is this doing anythig different to the above lines now?

Reply:
Not really I was thinking to leave it there to make sure that ghost jets are set up to the right px but I could probably do with a more simple test